README.md

```diff
@@ -112,6 +112,19 @@ In summary ...
 
 If you're using **System.Text.Json (STJ)**, then the setup described above will be sufficient, and *STJ* options/attributes will be automatically honored by the Swagger generator.
 
+If you find that the *STJ* options/attributes are not being honored, this may be because you are using a combination of minimal apis and mvc which have separate json options.
```

Comment:
```suggestion
If you find that the *STJ* options/attributes are not being honored, this may be because you are using a combination of Minimal APIs and MVC, which have separate JSON options.
```